### PR TITLE
added new blog post

### PR DESCRIPTION
**Pull request by andipavllo**

### CODE REVIEW

File: blog/posts/2015-04-22.markdown [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+author: andi
+date: 2015-04-22
+id: tutorial-customize-onsen-ui-themes
+title: "Tutorial: customize Onsen UI Themes"
+---
+
+![Onsen UI Components](/blog/content/images/2015/Apr/components.png)
+
+As probably most of our followers have noticed, Onsen UI has a wide set of premade themes, which cover a rich set of color combinations. But, how can the developer modify these themes, combining the preferred colors with the desired Onsen UI element? There are two ways to accomplish this.
+<br>
```
Comment (argelius) [inline blog/posts/2015-04-22.markdown]: Please remove <br> tags

**Comment by argelius** [comment]
Can you add some images where you have changed the colors? Maybe on with default colors and one with branding

File: blog/posts/2015-04-22.markdown [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+ But this approach has some limitations: the theme will be limited to the current version of Onsen UI. In case of future structure changes, it could not be compatible anymore with the new framework releases.
+<br>
+ Another limitation is that the themes are static and cannot be customized following the app behavior.
+
+ The dynamic customization of the themes is the solution of this kind of problems. Onsen UI themes are codified as Stylus, providing an efficient, dynamic, and expressive way to generate CSS.
+ <br>
+ Using the `.styl` file extension, Stylus accepts the standard CSS syntax, but it also accepts some other variations where brackets, colons, and semi-colons are all optional.
+ <br>
+ Modifying directly the `.styl` files provide the possibility to make Onsen UI layout dynamic, for example having different color combinations in different pages and offers an easier way to update the app layout once an Onsen UI upgrade is performed.
```
Comment (argelius) [inline blog/posts/2015-04-22.markdown]: This sounds a bit strange. Why would the developer want different colors on different pages?

Comment (andipavllo) [inline blog/posts/2015-04-22.markdown]: Well, I have seens some apps that have different elements color in some specific pages, like the settings one.

File: blog/posts/2015-04-22.markdown (hunk 1 above)
Comment (andipavllo) [inline blog/posts/2015-04-22.markdown]: Just from the preview or from all the document?

Comment (argelius) [inline blog/posts/2015-04-22.markdown]: Remove all of them.

**Pull request merged**